lib/govuk_publishing_components/presenters/machine_readable/government_service_schema.rb

```diff
@@ -0,0 +1,73 @@
+module GovukPublishingComponents
+  module Presenters
+    class GovernmentServiceSchema
+      attr_reader :page
+
+      def initialize(page)
+        @page = page
+      end
+
+      def structured_data
+        # http://schema.org/GovernmentService
+        {
+          "@context" => "http://schema.org",
+          "@type" => "GovernmentService",
+          "name" => page.title,
+          "serviceType" => page.description,
```

Comment:
Did we think this might be some sort of enumeration?  Something like https://standards.esd.org.uk/?uri=list%2Finteractions# (even though they're discontinued)

Reply:
As discussed, I'm going to remove `serviceType` for now because on reflection `description` is the more appropriate property to use for the description contained within our content schema for `transaction`s.